src/renderer/components/NewProjectModal/NewProjectView.tsx

```diff
@@ -0,0 +1,110 @@
+import { styled, Stack, Box, TextField } from '@mui/material';
+import CloseIcon from '@mui/icons-material/Close';
+import { ChangeEvent, useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { makeProjectWithoutMedia } from 'renderer/util';
+import { projectCreated } from 'renderer/store/actions';
+import CancelButton from '../CancelButton';
+import ActionButton from '../ActionButton';
+import colors from '../../colors';
+import { Project } from '../../../sharedTypes';
+import ModalTitle from '../ModalTitle';
+
+interface Props {
+  closeModal: () => void;
+  nextView: () => void;
+}
+
+const CustomStack = styled(Stack)`
+  width: 100%;
+`;
+
+const CustomTextField = styled(TextField)`
+  color: ${colors.white};
+`;
+
+const Container = styled(Box)`
+  background-color: ${colors.grey[700]};
+  height: 15vw;
+`;
+
+const NewProjectView = ({ closeModal, nextView }: Props) => {
+  const [projName, setProjName] = useState<string>('');
+  const [isAwaitingProjectName, setIsAwaitingProjectName] =
+    useState<boolean>(true);
+
+  const dispatch = useDispatch();
+
+  const setProjectInStore = async (project: Project) => {
+    dispatch(projectCreated(project));
+  };
+
+  const handleContinue = async () => {
+    const project = await makeProjectWithoutMedia(projName);
+    if (project === null) {
+      return;
+    }
+    setProjectInStore(project);
+    nextView();
+  };
+
+  const handleProjectNameInput = (
+    event: ChangeEvent<HTMLInputElement | HTMLTextAreaElement>
+  ) => {
+    setProjName(event.target.value);
+    if (event.target.value !== '') {
+      setIsAwaitingProjectName(false);
+    } else {
+      setIsAwaitingProjectName(true);
+    }
+  };
+
+  const continueButton = (
+    <ActionButton onClick={handleContinue} disabled={isAwaitingProjectName}>
+      Continue
+    </ActionButton>
+  );
+
+  const cancelButton = <CancelButton onClick={closeModal}>Cancel</CancelButton>;
```

Comment:
Do you know why these button labels are displaying in all caps?

Reply:
Nope

Reply:
Yeah it's a default part of the button styling - they have `text-transform` applied

Reply:
Figured this out - in themes.tsx under typography -> button add `textTransform: 'none',`

Reply:
done in: https://github.com/patrickbrett/mlvet/pull/38/commits/ab49c26e4721cc25c00fd971de2e26f92a3ed5cd